aeron-cluster/src/test/java/io/aeron/cluster/TestNode.java

```diff
@@ -240,6 +241,12 @@ public void onSessionMessage(
             final int length,
             final Header header)
         {
+            final String message = buffer.getStringWithoutLengthAscii(offset, length);
+            if (message.equals(TestMessages.REGISTER_TIMER))
+            {
+                cluster.scheduleTimer(1, cluster.timeMs() + 1_000);
```

Comment:
I've tried varying the expiry. At `10_000` milliseconds the test does not fail. It seems `TimerService.onTimerExpiry()` must be triggered on the leader for the test to fail.

Reply:
This should probably be lower to reduce "intermittency"